### ISSUE TITLE
liquibase diffChangeLog and generateChangeLog no longer have --include-schema option

### ISSUE THREAD

**Issue by lcshiung** [liquibase/liquibase#2108]
## Environment

**Liquibase Version**:
4.4.3

**Liquibase Integration & Version**: 
CLI

**Database Vendor & Version**:
PostgreSQL 13

**Operating System Type & Version**:
RHEL 7

## Description
liquibase diffChangeLog / generateChangeLog does not have --include-schema option.

## Steps To Reproduce
```
# liquibase diffChangeLog --help
Compare two databases to produce changesets and write them to a changelog file

Usage: liquibase diffChangelog [OPTIONS]

      --changelog-file=PARAM
                           [REQUIRED] Changelog file to write results
                           (liquibase.command.changelogFile OR liquibase.
                             command.diffChangelog.changelogFile)
                           (LIQUIBASE_COMMAND_CHANGELOG_FILE OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_CHANGELOG_FILE)
                           [deprecated: --changelogFile]

  -D=PARAM                 Pass a name/value pair for substitution in the
                             changelog(s)
                           Pass as -D<property.name>=<property.value>
                           [deprecated: set changelog properties in defaults
                             file or environment variables]
      --default-catalog-name=PARAM
                           The default catalog name to use for the database
                             connection
                           (liquibase.command.defaultCatalogName OR liquibase.
                             command.diffChangelog.defaultCatalogName)
                           (LIQUIBASE_COMMAND_DEFAULT_CATALOG_NAME OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_DEFAULT_CATALOG_NA
                             ME)
                           [deprecated: --defaultCatalogName]

      --default-schema-name=PARAM
                           The default schema name to use for the database
                             connection
                           (liquibase.command.defaultSchemaName OR liquibase.
                             command.diffChangelog.defaultSchemaName)
                           (LIQUIBASE_COMMAND_DEFAULT_SCHEMA_NAME OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_DEFAULT_SCHEMA_NAM
                             E)
                           [deprecated: --defaultSchemaName]

      --diff-types=PARAM   Types of objects to compare
                           (liquibase.command.diffTypes OR liquibase.command.
                             diffChangelog.diffTypes)
                           (LIQUIBASE_COMMAND_DIFF_TYPES OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_DIFF_TYPES)
                           [deprecated: --diffTypes]

      --driver=PARAM       The JDBC driver class
                           (liquibase.command.driver OR liquibase.command.
                             diffChangelog.driver)
                           (LIQUIBASE_COMMAND_DRIVER OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_DRIVER)

      --driver-properties-file=PARAM
                           The JDBC driver properties file
                           (liquibase.command.driverPropertiesFile OR liquibase.
                             command.diffChangelog.driverPropertiesFile)
                           (LIQUIBASE_COMMAND_DRIVER_PROPERTIES_FILE OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_DRIVER_PROPERTIES_
                             FILE)
                           [deprecated: --driverPropertiesFile]

      --exclude-objects=PARAM
                           Objects to exclude from diff
                           (liquibase.command.excludeObjects OR liquibase.
                             command.diffChangelog.excludeObjects)
                           (LIQUIBASE_COMMAND_EXCLUDE_OBJECTS OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_EXCLUDE_OBJECTS)
                           [deprecated: --excludeObjects]

  -h, --help               Show this help message and exit
      --include-objects=PARAM
                           Objects to include in diff
                           (liquibase.command.includeObjects OR liquibase.
                             command.diffChangelog.includeObjects)
                           (LIQUIBASE_COMMAND_INCLUDE_OBJECTS OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_INCLUDE_OBJECTS)
                           [deprecated: --includeObjects]

      --password=PARAM     The target database password
                           (liquibase.command.password OR liquibase.command.
                             diffChangelog.password)
                           (LIQUIBASE_COMMAND_PASSWORD OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_PASSWORD)

      --reference-default-catalog-name=PARAM
                           The reference default catalog name to use for the
                             database connection
                           (liquibase.command.referenceDefaultCatalogName OR
                             liquibase.command.diffChangelog.
                             referenceDefaultCatalogName)
                           (LIQUIBASE_COMMAND_REFERENCE_DEFAULT_CATALOG_NAME OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_REFERENCE_DEFAULT_
                             CATALOG_NAME)
                           [deprecated: --referenceDefaultCatalogName]

      --reference-default-schema-name=PARAM
                           The reference default schema name to use for the
                             database connection
                           (liquibase.command.referenceDefaultSchemaName OR
                             liquibase.command.diffChangelog.
                             referenceDefaultSchemaName)
                           (LIQUIBASE_COMMAND_REFERENCE_DEFAULT_SCHEMA_NAME OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_REFERENCE_DEFAULT_
                             SCHEMA_NAME)
                           [deprecated: --referenceDefaultSchemaName]

      --reference-password=PARAM
                           The reference database password
                           (liquibase.command.referencePassword OR liquibase.
                             command.diffChangelog.referencePassword)
                           (LIQUIBASE_COMMAND_REFERENCE_PASSWORD OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_REFERENCE_PASSWORD
                             )
                           [deprecated: --referencePassword]

      --reference-url=PARAM
                           [REQUIRED] The JDBC reference database connection URL
                           (liquibase.command.referenceUrl OR liquibase.command.
                             diffChangelog.referenceUrl)
                           (LIQUIBASE_COMMAND_REFERENCE_URL OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_REFERENCE_URL)
                           [deprecated: --referenceUrl]

      --reference-username=PARAM
                           The reference database username
                           (liquibase.command.referenceUsername OR liquibase.
                             command.diffChangelog.referenceUsername)
                           (LIQUIBASE_COMMAND_REFERENCE_USERNAME OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_REFERENCE_USERNAME
                             )
                           [deprecated: --referenceUsername]

      --schemas=PARAM      Schemas to include in diff
                           (liquibase.command.schemas OR liquibase.command.
                             diffChangelog.schemas)
                           (LIQUIBASE_COMMAND_SCHEMAS OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_SCHEMAS)

      --url=PARAM          [REQUIRED] The JDBC target database connection URL
                           (liquibase.command.url OR liquibase.command.
                             diffChangelog.url)
                           (LIQUIBASE_COMMAND_URL OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_URL)

      --username=PARAM     The target database username
                           (liquibase.command.username OR liquibase.command.
                             diffChangelog.username)
                           (LIQUIBASE_COMMAND_USERNAME OR
                             LIQUIBASE_COMMAND_DIFF_CHANGELOG_USERNAME)
```

## Actual Behavior
As described above.

## Expected/Desired Behavior
The --include-schema option (formerly --includeSchema) should be available.

## Additional Context
The issue was not presented in Liquibase version 4.3.5 (i.e. --includeSchema still works in that version)

**Comment by molivasdat** [comment]
Hi @lcshiung Thanks for bringing this parameter to our attention for the diffChangeLog command.  We add this to the list of issues that we are processing.

**Comment by nvoxland** [comment]
This is fixed by #2091
